sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaIO.java

```diff
@@ -500,6 +502,17 @@ public OutT apply(KafkaRecord<KeyT, ValueT> record) {
             return new KafkaConsumer<>(config);
           }
         };
+
+    @SuppressWarnings("unchecked")
+    @Override
+    public void populateDisplayData(DisplayData.Builder builder) {
+      super.populateDisplayData(builder);
+      builder.add(DisplayData.item("topics", Joiner.on(",").join(getTopics())));
+      for (Map.Entry<String, Object> conf : getConsumerConfig().entrySet()) {
```

Comment:
@aviemzur, I misread the code. getConsumerConfig() is what is our own map. Then we just need to exclude the internal configs (not a must, but better I think). I thought this was the map from 'ConsumerConfig()'.